cpp/tests/copying/concatenate_tests.cpp

```diff
@@ -703,7 +713,7 @@ TEST_F(ListsColumnTest, ConcatenateEmptyLists)
   }
 
   {
-    cudf::test::lists_column_wrapper<int> a{{LCW{}}};
+    cudf::test::lists_column_wrapper<int> a{LCW{}};
```

Comment:
I thought all these examples had all the nested brackets for a reason (nested lists?).

Reply:
As I noticed, if you write `{LCW{}}` then the compiler optimizes it out to just `LCW{}`. If you want to write nested list, you have to explicitly write `LCW{LCW{}}` (add the class name).

Reply:
The change was required to get past a runtime exception in `lists_column_wrapper`.  The extra `{}` were causing unexpected nested/depth levels. I worked with @nvdbaranec to work out the proper syntax for these.

Reply:
Oh yeah, I forgot that there is also inconsistency between the compilers' behavior!